.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,10 +4,9 @@ Please fill in the following checklist, removing items that do not apply.
 See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
 -->
 
-- **Version of the command being documented (if known):**  
-
 - [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
 - [ ] The page(s) have at most 8 examples.
 - [ ] The page description(s) have links to documentation or a homepage.
 - [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
 - [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
+- **Version of the command being documented (if known):**
```

Comment:
```suggestion
**Version of the command being documented (if known):**
```

Wait, what about to remove leading `-`?

Reply:
I like this change, but it's another discussion